.sqa/docker-compose.yml

```
@@ -0,0 +1,10 @@
+version: "3.6"
+
+services:
+  unit-testing:
+    image: "indigodatacloud/ci-images:python3.11"
+    hostname: "dcp-testing-host"
+    volumes:
+     - type: bind
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 5

(indentation)